packages/components/src/color-palette/stories/index.tsx

```diff
@@ -6,34 +12,24 @@ import { useState } from '@wordpress/element';
 /**
  * Internal dependencies
  */
-import ColorPalette from '../';
+import ColorPalette from '..';
 import Popover from '../../popover';
 import { Provider as SlotFillProvider } from '../../slot-fill';
+import type { ColorObject, PaletteObject } from '../types';
 
-const meta = {
+const meta: ComponentMeta< typeof ColorPalette > = {
 	title: 'Components/ColorPalette',
 	component: ColorPalette,
 	argTypes: {
+		// Removing the control because setting this prop without changing the
+		// format of the `colors` prop can break the component.
 		__experimentalHasMultipleOrigins: {
 			control: {
 				type: null,
 			},
 		},
-		__experimentalIsRenderedInSidebar: {
-			control: {
-				type: 'boolean',
-			},
-		},
-		clearable: {
-			control: {
-				type: 'boolean',
-			},
-		},
-		disableCustomColors: {
-			control: {
-				type: 'boolean',
-			},
-		},
+		onChange: { action: 'onChange', control: { type: null } },
```

Comment:
Let's also disable controls for the `as` prop:
```suggestion
		as: { control: { type: null } },
		onChange: { action: 'onChange', control: { type: null } },
```

Reply:
Thanks, committed in https://github.com/WordPress/gutenberg/pull/44632/commits/8001f145d7633d013b27d369e80d83bdc89b2864